src/components/ConnectionInterface.vue

```diff
@@ -140,6 +140,19 @@
         }
         this.$noty.success(`${config.name} deleted`)
       },
+      async duplicate(config) {
+        const duplicateConfig = Object.assign( Object.create( Object.getPrototypeOf(config)), config);
```

Comment:
Nice, please add a comment here about what this does (clone the ES6 class), because I think this is not obvious
```